src/components/search/SimpleSearch.tsx

```diff
@@ -72,97 +57,39 @@ const SimpleSearch: React.FC = () => {
-                  <Grid item>
-                    <Stack direction={'row'} spacing={0.5}>
-                      <Typography variant="h5">Search</Typography>
-                      {isGoldenOnly ? (
-                        <Typography
-                          variant="h5"
-                          sx={{ color: '#FBC02D', fontWeight: 700 }}
-                        >
-                          Golden
-                        </Typography>
-                      ) : (
-                        <Typography variant="h5" sx={{ fontWeight: 700 }}>
-                          Patient
-                        </Typography>
-                      )}
-
-                      <Typography
-                        variant="h5"
-                        sx={{ color: theme => theme.typography.h5 }}
-                      >
-                        Records
-                      </Typography>
-                    </Stack>
-                  </Grid>
-                  <Grid item sx={{ mb: 1 }}>
-                    <Stack direction={'row'} spacing={0.5}>
-                      <Typography variant="body2">
-                        Find info fast with these fixed fields or make your own
-                        search rules with
-                      </Typography>
-                      <Typography variant="body2">
-                        <Link href={'/search/custom'}>Custom Search</Link>
-                      </Typography>
-                    </Stack>
-                  </Grid>
-                </Grid>
-                <FieldArray name="search">
-                  {() => (
-                    <>
-                      {availableFields.map((field, index) => {
-                        const parameter = values.parameters[index]
-                        return (
-                          parameter && (
-                            <SearchRow
-                              field={field}
-                              parameter={parameter}
-                              onChange={handleChange}
-                              key={field.fieldName}
-                              index={index}
-                            />
-                          )
-                        )
-                      })}
-                    </>
-                  )}
-                </FieldArray>
-                <Grid item>
-                  <LocationLink
-                    to={`/search-results/${
-                      isGoldenOnly ? 'golden' : 'patient'
-                    }`}
-                    search={{ payload: values }}
-                    style={{ textDecoration: 'none' }}
-                  >
-                    <Button variant="contained">Search</Button>
-                  </LocationLink>
-                </Grid>
-              </Grid>
-            </Box>
-          </Form>
-        )}
-      </Formik>
+              <Button variant="contained">Search</Button>
+            </LocationLink>
```

Comment:
We should refactor this into the button component Lumiere has created.

Reply:
There is no Button component that can replace this code. We should create one.